src-db/database/sourcedata/AD_COLUMN.xml

```diff
@@ -1601,13 +1697,13 @@
 <!--D91BDA93612047D3B8CA82FCAEB81625-->  <AD_CLIENT_ID><![CDATA[0]]></AD_CLIENT_ID>
 <!--D91BDA93612047D3B8CA82FCAEB81625-->  <AD_ORG_ID><![CDATA[0]]></AD_ORG_ID>
 <!--D91BDA93612047D3B8CA82FCAEB81625-->  <ISACTIVE><![CDATA[Y]]></ISACTIVE>
-<!--D91BDA93612047D3B8CA82FCAEB81625-->  <NAME><![CDATA[Uninstall Dependency]]></NAME>
+<!--D91BDA93612047D3B8CA82FCAEB81625-->  <NAME><![CDATA[Uninstalldependency]]></NAME>
 <!--D91BDA93612047D3B8CA82FCAEB81625-->  <DESCRIPTION><![CDATA[Uninstall the selected dependency from the environment]]></DESCRIPTION>
 <!--D91BDA93612047D3B8CA82FCAEB81625-->  <HELP><![CDATA[Depending on the dependency format, the uninstallation will vary:
 * Dependency in JAR format: the process will simply delete the dependency register.
 * Dependency in Source format (downloaded from Module Management window or Local): apart from deleting the dependency, the process will also erase the dependency source folders from the environment.
 This process only deletes the selected module, not its dependencies.]]></HELP>
-<!--D91BDA93612047D3B8CA82FCAEB81625-->  <COLUMNNAME><![CDATA[Uninstalldependency]]></COLUMNNAME>
+<!--D91BDA93612047D3B8CA82FCAEB81625-->  <COLUMNNAME><![CDATA[Uninstall Dependency]]></COLUMNNAME>
```

Comment:
The column name **does** have to be a single word, so `uninstalldependency` is correct in this case